profiles/common/modules/features/multisite_business_indicators/multisite_business_indicators_community/tests/multisite_business_indicators_community.test

```diff
@@ -9,10 +9,12 @@
  */
 class BusinessIndicatorsCommunityQAWebTestCase extends DrupalWebTestCase {
   protected $profile = 'multisite_drupal_communities';
```

Comment:
Oops I'm sorry I wasn't clear in my previous review. The `{@inheritdoc}` documentation is for the `$profile` property, not for the class itself. The documentation for the class itself was good.

So it should look like this:

```
/**
 * Test Business Indicators ...
 */
class BusinessIndicatorBlaBlaQAWTC extends DWTC {

  /**
   * {@inheritdoc}
   */
  protected $profile = 'multisite_drupal_communities';
}
```

All methods and properties should be documented, but if they are inherited from a parent class or interface we should put `{@inheritdoc}` instead. The goal is that an IDE can then read the relevant documentation from the parent method. This will make sure that documentation is only written once.

Here is the relevant part in the coding standards: https://www.drupal.org/coding-standards/docs#inheritdoc